src/components/wayfinder/wayfinder.js

```diff
@@ -15,12 +15,38 @@ export default class Wayfinder extends React.Component {
 
   // TODO need to clear out and restart usersPath if user changes one of their answers
   clickHandler = event => {
+    const { usersPath } = this.state;
     const goto = event.target.dataset.goto;
+    const key = this.findKey(usersPath, goto);
+    let modifiedPath = [...usersPath];
+    console.log(usersPath);
+    modifiedPath.splice(usersPath.indexOf(key) + 1);
     this.setState(prevState => {
       return {
-        usersPath: prevState.usersPath.concat(goto)
+        usersPath: modifiedPath.concat(goto)
       };
     });
+    console.log(modifiedPath);
+  };
+
+  findKey = (path, child) => {
+    for (let key of path) {
```

Comment:
This could perhaps be a util function we use to generate info for the form as well.